packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -834,7 +901,7 @@ void main() {
 
     expect(editable, paints
       ..rrect(
-        rrect: RRect.fromRectAndRadius(
```

Comment:
nit: could you revert the indentation change?

Reply:
Done.